docs/notebooks/qaoa-transpiler.ipynb

```diff
@@ -0,0 +1,800 @@
e7cOerWrcuVK1fYtGkTxYsXNzuSiCH0DGUW9sQTT7Bs2TIOHjzIyy+/THJystmRHM6GDRvYs2cP/fr1MzuKiGQChQoVYs2aNbi5udGoUSNOnz5tdiQRQ2iGUli2bBl+fn7069ePL774wuw4DsXPz4+jR4+yZ88eLBaL2XFEJJM4duwYderUoUCBAmzYsAEvLy+zI4nYRDOUQosWLRg/fjzjxo3j66+/NjuOw/j9999ZunQpffv2VZkUEUOVLFmSNWvWcObMGZo2bcrly5fNjiRiExVKAeCtt96ib9++9O7dmxUrVpgdxyFMnDiRAgUK8PLLL5sdRUQyoaeffprVq1fz22+/0aJFC65evWp2JJGHplveclNycjJt27Zl3bp1bN68mcqVK5sdyTSXL1/miSeeoE+fPowYMcLsOCKSiUVGRtK4cWPq1KlDeHg42bNnNzuSyAPTDKXc5Orqyty5cylTpgzNmzfnr7/+MjuSaWbMmMG1a9fo1auX2VFEJJOrVasWS5YsYcOGDQQEBJCUlGR2JJEHphlK+Y9Tp05Rs2ZNChYsyMaNG8mdO7fZkTJUcnIyTz31FLVq1WLOnDlmxxGRLGLp0qW0bduWgIAAZs6ciYuL5nzEeehvq/xHkSJFWLZsGUeOHCEgICDLbSe0bNkyjh07Rt++fc2OIiJZSMuWLfnhhx+YPXs2QUFBaL5HnIlmKOWuVq1aRfPmzenVq1eWOnKwfv36JCYmsmXLFrOjiEgWFBwcTPfu3Xn33Xf59NNPtcuEOAU3swOI4/L19WXy5Mm88cYblC5dmj59+pgdye527drFhg0bWLhwodlRRCSL6tatG/Hx8fTr1w9PT0/ef/99syOJ3JcKpdxTz549OXLkCP369aNEiRK0atXK7Eh2NX78eIoWLUqbNm3MjiIiWVjfvn2Ji4vj/fffx9PTU6d1icNToZT7+vTTTzl27BgBAQFs3LiRatWqmR3JLs6ePcvcuXMZPnw4bm76pyEi5ho6dChxcXH0798fT09PunXrZnYkkbvSV025LxcXF77//nvq169Py5YtiY6OpmjRombHMtw333yDm5sb3bt3NzuKiAgWi4UxY8YQHx9Pjx49yJUrFx07djQ7lsgdaVGOpNvff/+Nj48PefLkYfPmzeTJk8fsSIZJSEigePHitG3bVsdPiohDSUlJoWvXrsydO5dFixbRsmVLsyOJ/Ie2DZJ0e+SRR1ixYgUnTpygffv2mWrz3QULFvD3339niYVHIuJcXFxcCA4OplWrVrz00kusXbvW7Egi/6EZSnlga9as4cUXX6Rbt25MmTLF6be0sFqtVKtWjUceeYT//e9/ZscREbmjhIQEWrduzcaNG4mIiKB27dpmRxK5STOU8sAaNWrEN998w9SpUxk3bpzZcWy2efNmfvnlF62iFBGHlj17dkJDQ6levTrNmjVj586dZkcSuUkzlPLQhgwZwpgxYwgJCaFt27Zmx3lo7dq14+DBg+zfv9/pZ1tFJPO7fPkyjRo14tixY/z000+UL1/e7EgiKpTy8FJSUggMDGTJkiVs2LCBGjVqmB3pgR0/fpxSpUrd3MBdRMQZxMbGUq9ePc6fP8+mTZt48sknzY4kWZwKpdjk+vXrNGzYkCNHjhAdHU2JEiXMjvRA3nnnHWbMmMHJkyfJlSuX2XFERNLtzJkz1K1bl6SkJDZt2sTjjz9udiTJwvQMpdgkR44chIWFkTt3bpo3b87FixfNjpRucXFxTJ8+nddff11lUkSczqOPPsqaNWtITk6mUaNGnD171uxIkoWpUIrNChUqxIoVKzh9+jT+/v7cuHHD7EjpMmvWLOLj4wkKCjI7iojIQylWrBhr1qzhwoUL+Pr6OtU39ZK5qFCKIcqWLcuiRYvYuHEjb775Jo7+JEVKSgoTJkygXbt2mfLUHxHJOp566inWrFnDiRMnePHFF4mPjzc7kmRBKpRimHr16jF9+nSCg4P59NNPzY5zT//73//47bfftFWQiGQK3t7erFy5kv379+Pn58f169fNjiRZjBbliOE++ugjPv74YxYsWED79u3NjnNHjRs35vLly0RFRWmrIBHJNDZu3EjTpk1p2LAhixYtwt3d3exIkkWoUIrhrFYrnTt35scff2T9+vXUqlXL7Ej/sm/fPipWrMjcuXMJCAgwO46IiKFWrVpFy5Ytadu2LXPmzMHV1dXsSJIFqFCKXSQkJNC4cWMOHjxIVFQUpUqVMjvSTT169GDFihUcP35c372LSKa0aNEi2rdvT5cuXZg2bRouLnrCTexLf8PELrJnz87ixYvJnz8/zZs35/z582ZHAiAmJobZs2cTFBSkMikimVbbtm357rvvmDFjBv3793f4hZLi/FQoxW68vLxYsWIFMTExtG3blsTERLMj8e233wLw+uuvm5xERMS+OnXqxNdff82ECRP44IMPzI4jmZwKpdhV6dKlCQsLIzIykh49epj6XXJiYiKTJ0+mU6dOFCxY0LQcIiIZ5c033+Szzz5jxIgRfPbZZ2bHkUzMzewAkvnVqVOHmTNnEhgYSOnSpXn//fdNyRESEsKpU6fo06ePKdcXETHDu+++S1xcHO+99x65c+emV69eZkeSTEiFUjJEQEAAR48eZdiwYTz55JO8/PLLGXp9q9XKV199RaNGjfD29s7Qa4uImO3jjz8mLi6OoKAgcufOTefOnc2OJJmMCqVkmCFDhnDkyBFee+01ihUrxvPPP59h146KimLbtm0sW7Ysw64pIuIoLBYL48aNIz4+nq5du5IrVy7atWtndizJRLRtkGSoxMREmjZtyu7du4mMjKRMmTIPNY4VK+c4x1Wukkgi2ciGBx4UohAW/rtReYcOHfjll1/49ddftX2GiGRZycnJvPLKK4SGhhIeHs6LL75odiTJJFQoJcNduHCB2rVrk5SURGRkZLoWyMQQwzrWsYMdbGUrO9hBHHH/eT9PPKlGNWpQg2pUowENuHbyGiVLluSrr
```

Comment:
On the plot above you should add some labels, e.g. Probability for the y axis and sth like "bit-string value" on the x-axis.

Reply:
Also, one thing that we did here: was color the maximum cuts https://github.com/qiskit-community/qopt-best-practices/blob/main/demos/qiskit_patterns/2_qiskit_patterns.ipynb